recipes/cascade/bld.bat

```diff
@@ -0,0 +1,14 @@
+mkdir build
+cd build
+
+cmake ^
+    -G "Visual Studio 16 2019" -A x64 ^
+    -DPython3_EXECUTABLE=%PREFIX%\python.exe ^
+    -DCMAKE_INSTALL_PREFIX=%LIBRARY_PREFIX% ^
+    -DCMAKE_PREFIX_PATH=%LIBRARY_PREFIX% ^
+    -DCASCADE_BUILD_TESTS=no ^
+    -DBoost_NO_BOOST_CMAKE=ON ^
+    -DCASCADE_BUILD_PYTHON_BINDINGS=yes ^
+    ..
+
+cmake --build . --config Release --target install
```

Comment:
Is there a `make check` or ctest that we can run here and in the .sh build file?

Reply:
Activating the -DCASCADE_BUILD_TESTS C++ tests ... this will substantially lengthen the build and tests though and they are already run on the github repo on equivalent environments. If possible I would avoid.

Reply:
We highly encourage folks to run all the tests if the CI time allows it. Or a subset if not. But it is compiled code so testing it in our CI is not the same thing as upstream.

Reply:
ok added all tests ...